plugins/services/src/js/utils/__tests__/TaskTableUtil-test.js

```diff
@@ -9,13 +9,14 @@ describe('TaskTableUtil', function () {
   beforeEach(function () {
     this.getServiceHealth = MarathonStore.getServiceHealth;
     MarathonStore.getServiceHealth = function (prop) {
-      return this[prop].get('health');
+      return this[prop]['health'];
```

Comment:
I don't see us using tasks as `Items` anywhere. We do this for Services.
Either we should make the effort and make everywhere we use tasks to structs or we keep this change to do as we do elsewhere when working with tasks.
This spawns a larger discussion about the future of structs in our code base and what the plan is to get where we want.
I am ok with this change as it follows what we do elsewhere.